src/runtime/composables/useClientStripe.ts

```diff
@@ -1,4 +1,4 @@
-import { useNuxtApp } from '#app'
+import { useNuxtApp } from "#imports"

```

Comment:
the doc is speaking about this import instead of #app 

https://nuxt.com/docs/guide/going-further/modules#runtime-directory

Reply:
I can't use the composable locally don't know why